components/style/properties.mako.rs

```diff
@@ -55,8 +55,14 @@ class Keyword(object):
         self.name = name
         self.values = values
         self.gecko_constant_prefix = gecko_constant_prefix or "NS_STYLE_" + self.name.upper().replace("-", "_")
-        self.extra_gecko_values = extra_gecko_values or []
-        self.extra_servo_values = extra_servo_values or []
+        if extra_gecko_values is None:
+            self.extra_gecko_values = []
+        else:
+            self.extra_gecko_values = extra_gecko_values.split()
```

Comment:
This can be: |self.extra_gecko_values = (extra_gecko_values or "").split()|, which will let us remove the conditional.

Same things for extra_servo_values below.